keyboards/deltasplit75/v2/v2.h

```diff
@@ -0,0 +1,37 @@
+#ifndef V2_H
+#define V2_H
+
+#include "deltasplit75.h"
+
+//void promicro_bootloader_jmp(bool program);
+#include "quantum.h"
+
+//void promicro_bootloader_jmp(bool program);
+//matrix is defined in a weird way here; the layout on both sides are asymmetrical, but the "matrix" is symmetrical but with empty gaps
+//the last column is defined as a separate row because the firmware currently doesnt support more than 8 columns (this layout has 9 columns per side) K45 and K110 are the Bs on both sides; K53 and K106 are extra keys for ISO
+#define KEYMAP( \
```

Comment:
One way to fix the error would be to define two KEYMAPs, `KEYMAP_V2` and `KEYMAP_PROTOSPLIT` with different numbers of arguments (88 and 90 it looks like), and use those in the keymaps. 

The other (possibly easier) way would be to just accept 90 in both, and always have `KC_NO` in the keymaps that don't use them.